recipe/meta.yaml

```diff
@@ -8,15 +8,15 @@ package:
 
 source:
   url: https://cache.ruby-lang.org/pub/ruby/{{ major_minor }}/ruby-{{ version }}.tar.gz
-  sha256: 61843112389f02b735428b53bb64cf988ad9fb81858b8248e22e57336f24a83e
+  sha256: a3d55879a0dfab1d7141fdf10d22a07dbf8e5cdc4415da1bde06127d5cc3c7b6
   patches:
     # Patches/work-arounds for passing make check
     - disable-backtrace-with-lines.patch
     - fix-isinf-redefinition-msvc.patch  # [win]
     - 0001-Dont-t-add-SDKROOT-to-vals.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Do we need to increment the build number even though we are changing the version number?

Reply:
Um, nope, that's a mistake.

Reply:
Fixed, thanks for spotting that!